lib/todo_web/controllers/user_reset_password_controller.ex

```diff
@@ -3,10 +3,11 @@ defmodule TodoWeb.UserResetPasswordController do
 
   alias Todo.Accounts
 
+  plug :put_layout, {TodoWeb.LayoutView, "auth_form.html"}
   plug :get_user_by_reset_password_token when action in [:edit, :update]
 
   def new(conn, _params) do
-    render(conn, "new.html")
+    render(conn, "new.html", error_message: nil, title: "Forgot your password?")
```

Comment:
Sorry. I might be missing something, but why do we need to pass in `error_message` here?

Reply:
I'm not sure. I had just copied it from the login controller. I removed it.

Reply:
Ah, thanks for pointing that out. I see that the login controller does use `error_message` in the template when an authentication error occurs, so that would explain the difference in behavior here.